artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PagedMessageImpl.java

```diff
@@ -47,6 +47,8 @@
 
    private long transactionID = 0;
 
+   private int storedSize = 0;
```

Comment:
If it cannot change anymore, I would make it `final` instead

Reply:
I didn't think it was final.. I thought I was going to have a setter when I wrote this.. changing it.